### PR TITLE
fix broken cluster/centos and enhance the style

### PR DESCRIPTION
**Pull request by xiaoping378**
replace the PR #33818

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34002)

### CODE REVIEW

**Comment by xiaoping378** [comment]
Hi @roberthbailey 
i must sign up to contribute to this project as an employee for cla/linuxfoundation ?

**Comment by roberthbailey** [comment]
@zmerlynn can you take a look at this today?

**Comment by mikedanese** [comment]
@k8s-bot bazel test this, issue kubernetes/test-infra#1152

**Comment by rmmh** [comment]
@k8s-bot test this

**Comment by rmmh** [comment]
@xiaoping378 looks like one of the copyrights is wrong

> 
> I1208 10:15:21.421] Verifying hack/make-rules/../../hack/verify-boilerplate.sh
> I1208 10:15:22.567] Boilerplate header is wrong for: /go/src/k8s.io/kubernetes/cluster/centos/master/scripts/flannel.sh
> I1208 10:15:22.569] FAILED   hack/make-rules/../../hack/verify-boilerplate.sh	1s
>

**Comment by xiaoping378** [comment]
@rmmh 
should I run hack/update-generated-docs.sh ?
and add a commit ?

**Comment by rmmh** [comment]
No. hack/verify-boilerplate.sh should show tell you what's wrong-- I think you might just have an extra line (and the wrong year) in the copyright in flannel.sh.

**Comment by rmmh** [comment]
feel free to squash your commits

File: cluster/centos/master/scripts/flannel.sh [hunk 1]
```diff
@@ -60,5 +60,6 @@ while true; do
 done
 wait
 
+systemctl enable flannel
```
Comment (xiaoping378) [inline cluster/centos/master/scripts/flannel.sh]: yes, need it.
ok i will squash the two commits

**Comment by xiaoping378** [comment]
Just run
```
git rebase -i HEAD~2
git push -f 
```

**Pull request merged**